### ISSUE TITLE
Sample android app link on the website is broken. 

### ISSUE THREAD

**Issue by VisheshVadhera** [square/tape#134]
The sample android app link on the website directs the users to [this](https://github.com/square/tape/tree/master/tape-sample) which returns a 404.

**Comment by NightlyNexus** [comment]
Yeah, it should be https://github.com/square/tape/tree/6107873105376eec3b8d0a75b4a41a580a6a30ba/tape-sample
It's probably not worth updating the site now, but there's the code if you want it.
